docs/src/languages/php/extensions.md

```diff
@@ -60,110 +61,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| ioncube          |     |     |     | *   | *   | *   |     |     |     |
+| json             |     |     | *   | *   | *   | *   | *   | *   | *   |
+| ldap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mailparse        |     |     |     | *   | *   | *   |     | *   |     |
+| mbstring         |     |     |     | *   | *   | *   | *   | *   | *   |
+| mcrypt           | *   | *   | *   | *   | *   |     |     |     |     |
+| memcache         | *   | *   | *   |     |     |     |     |     |     |
+| memcached        | *   | *   | *   | *   | *   | *   |     | *   |     |
+| mongo            | *   | *   | *   |     |     |     |     |     |     |
+| mongodb          |     |     |     | *   | *   | *   | *   | *   |     |
+| msgpack          |     |     | *   | *   | *   | *   |     | *   |     |
+| mssql            | *   | *   | *   |     |     |     |     |     |     |
+| mysql            | *   | *   | *   |     |     |     |     |     |     |
+| mysqli           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mysqlnd          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| newrelic         |     |     | *   | *   | *   | *   | *   | *   | *   |
+| oauth            |     |     |     | *   | *   | *   | *   | *   |     |
+| odbc             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| opcache          |     | *   | *   | *   | *   | *   | *   | *   | *   |
+| openssl          |     |     |     |     |     |     |     |     |     |
+| pdo              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_dblib        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_firebird     | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_mysql        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_odbc         | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_pgsql        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_sqlite       | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_sqlsrv       |     |     |     | *   | *   | *   |     | *   |     |
+| http             |     |     | *   |     |     |     |     | *   |     |
+| pgsql            | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| phar             |     |     |     | *   | *   | *   | *   | *   | *   |
+| pinba            | *   | *   | *   |     |     |     |     |     |     |
+| posix            |     |     |     | *   | *   | *   | *   | *   | *   |
+| propro           |     |     | *   |     |     |     |     |     |     |
+| pspell           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pthreads         |     |     |     |     | *   |     |     |     |     |
+| raphf            |     |     | *   |     |     |     |     | *   | *   |
+| readline         | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| recode           | *   | *   | *   | *   | *   | *   | *   | *   |     |
```

Comment:
Pourquoi?

Reply:
Look like recode has been removed in php7.4 https://www.php.net/manual/en/migration74.removed-extensions.php and we don't have it for php7.4 too ref 244f4a0e